docs/core-concepts/dotnet-core.md

```diff
@@ -0,0 +1,145 @@
+
+Composition
+===========
+
+.NET Core is composed of a few parts:
+
+- A [.NET runtime](https://github.com/dotnet/coreclr), which provides a type system, assembly loading, a garbage collector, native interop and other basic services. 
+- A set of [framework libraries]((https://github.com/dotnet/corefx)), which provide primitive data types, app composition types and fundamental utilities. 
+- The 'dotnet' app host, which is used to launched .NET Core apps. It selects the runtime and hosts the runtime, provides an assembly loading policy and launches the app. The same host is also used to launch SDK tools in much the same way.
+- A [set of SDK tools](https://github.com/dotnet/cli) and language compilers that enable the base developer experience, available in the [.NET Core SDK](http://dotnet.github.io/docs/core-concepts/core-sdk).
+
+Languages
+---------
+
+The C# languages (F# and VB are coming) can be used to write applications and libraries for .NET Core. The compilers run on .NET Core, enabling you to develop for .NET Core anywhere it runs. In general, you will not use the compilers directly, but indirectly by using the SDK tools.
+
+The C# Roslyn compiler and the .NET Core tools have been integrated into several text editors and IDEs, including Visual Studio, Visual Studio Code, Sublime Text and Vim, making .NET Core development an option in your favorite coding environment and OS. This integration is provided, in part, by the good folks of the [OmniSharp project](http://www.omnisharp.net/).
+
+.NET APIs and Compatibility
+---------------------------
+
+.NET Core can be thought of as a cross-platform version of the .NET Framework, at the layer of the .NET Framework Base Class Libraries (BCL). It provides a subset of the APIs that are available in the .NET Framework or Mono/Xamarin. In some cases, types have been subseted (some members are not available or have been moved).
+
+The focus of .NET Core 1.0 has been "bringup" of the platform. Post .NET Core 1.0, there will be a focus on compatibility to make it easier to take advantage of .NET Core with existing .NET Framework and Mono codebases.
+
+Relationship to the .NET Standard Library
+-----------------------------------------
+
+The [.NET Standard Library](../concepts/standard-library.md) is an API spec that describes the consistent set of .NET APIs that developers can expect in each .NET implementation. .NET implementations need to implement this spec in order to be considered .NET Standard Library compliant and to support libraries that target the .NET Standard Library. 
+
+.NET Core implements the .NET Standard Library, and therefore supports .NET Standard Libraries.
+
+Workloads
+---------
+
+.NET Core includes a single application model: console apps. That's useful for tools, local services and console games. Additional application models build on top of .NET Core, such as: 
+
+- [ASP.NET Core](http://asp.net)
+- [Windows 10 UWP](https://developer.microsoft.com/windows)
```

Comment:
Channeling @mairaw here: explode the TLAs first time they are used in a text. :)